### PR TITLE
Push the data elastic search after getting the results

### PR DESCRIPTION
**Pull request by athiruma**
1. Push the results to elastic search, without explicitly calling the upload_es method from s3.
2. Set the default index: cloud-governance-policy-es-index.
3. Check the elastic search connection before pushing the data

### CODE REVIEW

**Review by ebattat — COMMENTED** [review]
Are we going to upload every transaction?

**Comment by athiruma** [comment]
> Are we going to upload every transaction?

yes.

**Comment by ebattat** [comment]
@athiruma, what will be if elastic search will be down for maintenance, all the data will be gone ?

**Comment by athiruma** [comment]
> @athiruma, what will be if elastic search will be down for maintenance, all the data will be gone ?

We store this data in s3 as a backup.

File: cloud_governance/common/elasticsearch/elasticsearch_operations.py [hunk 1]
```diff
@@ -34,8 +35,10 @@ def __init__(self, es_host: str = None, es_port: str = None, region: str = '', b
         self.__es_port = es_port if es_port else self.__environment_variables_dict.get('es_port')
         self.__region = region
         self.__timeout = int(self.__environment_variables_dict.get('ES_TIMEOUT')) if self.__environment_variables_dict.get('ES_TIMEOUT') else timeout
-        self.__es = Elasticsearch([{'host': self.__es_host, 'port': self.__es_port}], timeout=self.__timeout, max_retries=2)
-
+        try:
+            self.__es = Elasticsearch([{'host': self.__es_host, 'port': self.__es_port}], timeout=self.__timeout, max_retries=2)
+        except:
+            pass
```
Comment (ebattat) [inline cloud_governance/common/elasticsearch/elasticsearch_operations.py:41]: is it ok to put pass here, dont we want to raise an error ?

Comment (athiruma) [reply cloud_governance/common/elasticsearch/elasticsearch_operations.py:41]: Even though elastic search fails. We need to send data to s3.  That's the reason I wrote pass there

**Review by ebattat — APPROVED** [review]
/lgtm

**Pull request merged**